eureka2-ext/eureka2-overrides-aws/src/main/java/com/netflix/eureka2/server/service/overrides/OverridesCodec.java

```diff
@@ -0,0 +1,20 @@
+package com.netflix.eureka2.server.service.overrides;
+
+import com.netflix.eureka2.codec.json.EurekaJsonCodec;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author David Liu
+ */
+public final class OverridesCodec {
```

Comment:
Agree. This is currently in an awkward place here but it's also slightly different in that it's not strictly for the eureka transport layer. Some small reorg of the transport and the codecs may be necessary.